apps/src/levelbuilder/lesson-editor/LessonEditor.jsx

```diff
@@ -272,6 +275,18 @@ class LessonEditor extends Component {
               </p>
             </HelpTip>
           </label>
+          <label>
+            Background
```

Comment:
Could we include a small note here in the UI explaining what this does?

Reply:
Good call.  Will add a `HelpTip`.